src/data/move.ts

```diff
@@ -4516,6 +4516,7 @@ export class ForceSwitchOutAttr extends MoveEffectAttr {
 	  if (switchOutTarget instanceof PlayerPokemon) {
 	  	if (switchOutTarget.hp) {
```

Comment:
I know it's not your code but can we please add readability to this check?

```suggestion
	  	if (switchOutTarget.hp > 0) {
```